src/Symfony/Component/Serializer/Tests/Encoder/XmlEncoderTest.php

```diff
@@ -979,7 +922,7 @@ public function testEncodeWithoutPi()
 
         $expected = '<response/>';
 
-        $this->assertEquals($expected, $encoder->encode([], 'xml'));
+        static::assertEquals($expected, $encoder->encode([], 'xml'));
```

Comment:
As we're using `$this->` everywhere, this change should be part of PHP-CS-Fixer so that it can be enforced. Until then, I propose to revert such changes.